swimlane/core/resources/record.py

```diff
@@ -46,7 +47,7 @@ def __init__(self, app, raw):
             # Combine app acronym + trackingId instead of using trackingFull raw
             # for guaranteed value (not available through report results)
             self.tracking_id = '-'.join([
-                self._app.acronym,
+                self.__app.acronym,
```

Comment:
Change these internal references to the now public `.app` prop wherever you can.